server/controller/FlightController.js

```diff
@@ -334,6 +334,8 @@ router.get(
       const { airspaceViolation } = await runChecksStartCalculationsStoreFixes(
         flight,
         null,
+
+        // TODO: Should the recalculation really always skip all checks?
```

Comment:
Only triggert by admin... so why not?
Is needed because a flight is only modifiable within the first 14 days... rerun could be performed after that threshold

Reply:
Ja, aber sollten dann auch alle anderen checks geskipped werden? Airspace, G-Record, Midflight?

Reply:
Midflight wird auch geskipped.
G-Record wird bei rerun aktuell grundsätzlich nicht mehr geprüft; Das IGC hat sich ja auch nicht verändert.
Airspaces wären dann eigentlich auch überflüssig.

Reply:
Airspace würde ich nichts kippen. Vllt hat ja irgendwas nicht gestimmt und man macht deshalb die recalc.